web/satellite/src/components/accessGrants/ConfirmDeletePopup2.vue

```diff
@@ -0,0 +1,266 @@
+                    height="44px"
+                    :on-press="onCancelClick"
+                    is-white="true"
+                    :is-disabled="isLoading"
+                />
+                <VButton
+                    label="Delete Access"
+                    width="150px"
+                    height="44px"
+                    :on-press="onDeleteClick"
+                    :is-disabled="isLoading || confirmedInput !== selectedAccessGrants[0].name"
+                    is-solid-delete="true"
+                    has-trash-icon="true"
+                />
+            </div>
+            <div class="confirm-delete__container__close-cross-container" @click="onCancelClick">
+                <CloseCrossIcon />
+            </div>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+import { MetaUtils } from '@/utils/meta';
+
+import VButton from '@/components/common/VButton.vue';
+import HeaderlessInput from '@/components/common/HeaderlessInput.vue'
+
+import CloseCrossIcon from '@/../static/images/common/closeCross.svg';
+
+import { ACCESS_GRANTS_ACTIONS } from '@/store/modules/accessGrants';
+import { AccessGrant } from '@/types/accessGrants';
+
+// @vue/component
+@Component({
+    components: {
+        VButton,
+        HeaderlessInput,
+        CloseCrossIcon,
+    },
+})
+export default class ConfirmDeletePopup extends Vue {
+    private FIRST_PAGE = 1;
+    private isLoading = false;
+    private confirmedInput = '';
+
+    /**
+     * sets Comfirmed Input property to the given value.
+     */
+    public setConfirmedInput(value: string): void {
+        this.confirmedInput = value;
+    }
+
+    /**
+     * Deletes selected access grants, fetches updated list and closes popup.
+     */
+    public async onDeleteClick(): Promise<void> {
+        if (this.isLoading) return;
+
+        this.isLoading = true;
+
+        try {
+            await this.$store.dispatch(ACCESS_GRANTS_ACTIONS.DELETE);
+            await this.$notify.success(`Access Grant deleted successfully`);
+        } catch (error) {
+            await this.$notify.error(error.message);
+        }
+
+        try {
+            await this.$store.dispatch(ACCESS_GRANTS_ACTIONS.FETCH, this.FIRST_PAGE);
+            await this.$store.dispatch(ACCESS_GRANTS_ACTIONS.CLEAR_SELECTION);
+        } catch (error) {
+            await this.$notify.error(`Unable to fetch Access Grants. ${error.message}`);
+        }
+
+        this.$emit('reset-pagination');
+        this.isLoading = false;
+        this.onCancelClick();
+    }
+
+    /**
+     * Closes popup
+     */
+    public onCancelClick(): void {
+        this.$emit('close');
+    }
+    /**
+     * Checks for new access grant flag
+     */ 
+    public get isNewAccessGrantFlow(): boolean {
+        const isNewAccessGrantFlow = MetaUtils.getMetaContent('new-access-grant-flow');
+        return isNewAccessGrantFlow === "true";
```

Comment:
we can Boolean(var)

Reply:
I removed the whole method as the feature flag is being controlled at the parent level and has no use on this component.